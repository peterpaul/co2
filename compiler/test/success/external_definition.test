package compiler.test;

["stdio.h"] int printf(char * format, ...);

["compiler/test/success/include.h"] int test_var;
["compiler/test/success/include.h"] int test_fun(int a, int b);

int main(int argc, char ** argv)
{
	printf("Hello %s\n", "World");
	test_var = 5;
	printf("5 + 4 = %d\n", test_fun(test_var, 4));
	return 0;
}