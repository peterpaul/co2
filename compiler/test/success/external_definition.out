Hello World
5 + 4 = 9
